### PR TITLE
{64} Style the Product Details Page using CSS

### PR DESCRIPTION
**Pull request by Tocyn**
Screen Shot 2020-11-10 at 12 36 03 PM
Screen Shot 2020-11-10 at 12 47 25 PM
Screen Shot 2020-11-10 at 1 44 57 PM

### CODE REVIEW

**Comment by Tocyn** [comment]
Screen Shot 2020-11-10 at 2 12 17 PM
Screen Shot 2020-11-10 at 2 12 31 PM
Screen Shot 2020-11-10 at 2 12 56 PM

**Comment by adaugochi** [comment]
The content on the body section should take more place around it. It looks too tiny and compressed. Meanwhile there are spaces left and right

**Comment by Tocyn** [comment]
Screen Shot 2020-11-11 at 10 57 49 AM
Screen Shot 2020-11-11 at 10 58 18 AM

**Comment by Tocyn** [comment]
Screen Shot 2020-11-11 at 11 16 45 AM

**Comment by Ireti203** [comment]
The input field for email address on the footer (mobile screen) seem invisible. Can you correct this please.

**Review by Ireti203 — CHANGES_REQUESTED** [review]
Kindly give this css file a better name. Dont forget to effect the change in your html file. 

Weldone @Tocyn

**Comment by Tocyn** [comment]
Screen Shot 2020-11-15 at 8 41 11 PM
Screen Shot 2020-11-15 at 8 16 02 PM

**Comment by Tocyn** [comment]
> The input field for email address on the footer (mobile screen) seem invisible. Can you correct this please.

Done. kindly review

**Comment by adaugochi** [comment]
Kindly style your input field and button

**Comment by Tocyn** [comment]
Screen Shot 2020-11-17 at 12 19 49 PM
Screen Shot 2020-11-17 at 12 20 30 PM
Screen Shot 2020-11-17 at 12 23 02 PM

File: submissions/oluwatosinOke/html/productPage.html [hunk 1]
```diff
@@ -1,41 +1,100 @@
...
-            <li> RAM:4GB </li>
-            <li> Camera: Front and Rear </li>
-        </ol>
-        <strong> Similar Product from seller: </strong> 
-        <ol>
-            <li> Samsung A21 </li>
-            <li> Samsung A10s </li>
-            <li> Samsung A51 </li>
-        </ol>
+        <form>
```
Comment (ToluAkin) [inline submissions/oluwatosinOke/html/productPage.html:11]: @Tocyn Your product page should not be wrapped in a form tag. It isn't a form.

File: submissions/oluwatosinOke/html/productPage.html [hunk 2]
```diff
@@ -1,41 +1,100 @@
...
+                <div class="container">
+                    <div class="left-column">
+                        <img src ="./images/samsung-galaxy-a20.jpg">   
+                    </div>
+                    <div class="right-column">
+                        <div class="product-information">
+                            <h3>Galaxy A20</h3>
+                            <p>Brand: Samsung</p>
+                            <label for="quantity">Quantity: </label>
+                            <input min="1" type="number" id="quantity" name="quantity" value="0" /><br>
```
Comment (ToluAkin) [inline submissions/oluwatosinOke/html/productPage.html:36]: Remove the `<br>` here and style the input field with css @Tocyn

File: submissions/oluwatosinOke/html/productPage.html [hunk 3]
```diff
@@ -1,41 +1,100 @@
...
+                            <p>Price: &#x20A6;75,000</p>
+                            <p>shipping cost: &#x20A6;1,000</p>
+                            <p>Total Cost: &#x20A6;76,000</p>
+                            <input type="button" value="BUY NOW">
+                        </div>
+                    </div>
+                    <div class="description">
+                        <h3> Product Description: <br>
```
Comment (ToluAkin) [inline submissions/oluwatosinOke/html/productPage.html:44]: Remove the `<br>` here and style the input field with css @Tocyn

File: submissions/oluwatosinOke/html/productPage.html [hunk 4]
```diff
@@ -1,41 +1,100 @@
...
+                        This new samsung has a 6.4" LCD display, 32GB Storage,4GB RAM, 13MP front and rear Camera </h3>
+                    </div>
+                    <div class="bottom-column"> 
+                        <h3>RELATED PRODUCT</h3>
+                        <ul>
+                            <li>
+                                <figure>
+                                    <img src = "./images/iPhone-11.jpg">
+                                    <figcaption>iphone-11 <br> price: &#x20A6;120,000</figcaption>
```
Comment (ToluAkin) [inline submissions/oluwatosinOke/html/productPage.html:53]: Put them in 2 separate paragraph tags instead. @Tocyn

File: submissions/oluwatosinOke/html/productPage.html [hunk 5]
```diff
@@ -1,41 +1,100 @@
...
+                            <input type="button" value="SUBMIT">
+                        </div>
+                        <div class="footer-right">
+                            <h3>Connect With Us</h3>
+                            <a href="http//:www.facebook.com"><img src="../html/images/facebook-logo.png" alt=""></a>
+                            <a href="http//:www.twiiter.com"><img src="../html/images/Twitter-logo.png" alt=""></a>
+                            <a href="http//:www.gmail.com"><img src="../html/images/Gmail-logo.png" alt=""></a>
+                            <a href="http//:www.youtube.com"><img src="../html/images/youtube-logo.png" alt=""></a>
+                        </div>
+                        <p> Copyright &copy; 2020 myfavoriteonlinestore.com<br>
```
Comment (ToluAkin) [inline submissions/oluwatosinOke/html/productPage.html:92]: Put them in 2 separate paragraph tags instead. @Tocyn

**Review by ToluAkin — CHANGES_REQUESTED** [review]
Try as much as possible to avoid using `br` in your html. Well done!

File: submissions/oluwatosinOke/html/productPage.html [hunk 6]
```diff
@@ -1,41 +1,103 @@
...
+                            <label for="quantity">Quantity: </label>
+                            <input min="1" type="number" id="quantity" name="quantity" value="0"/>
+                            <p>Price: &#x20A6;75,000</p>
+                            <p>shipping cost: &#x20A6;1,000</p>
+                            <p>Total Cost: &#x20A6;76,000</p>
+                            <input type="button" value="BUY NOW">
+                        </div>
+                    </div>
+                    <div class="description">
+                        <h3> <p>Product Description:</p>
```
Comment (ToluAkin) [inline submissions/oluwatosinOke/html/productPage.html:43]: Either you use the p tag or the h3 tag. You can't have p nested on h3 @Tocyn

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Review by adaugochi — APPROVED** [review]

**Comment by adaugochi** [comment]
Kindly use `[]` not `{}` @Tocyn  on your PR title.

**Pull request merged**